src/data/drinkRecipes/pumpkinBuck.ts

```diff
@@ -0,0 +1,117 @@
+import type { DrinkRecipe } from "types/DrinkRecipe"
+import { DrinkRecipeIngredientType } from "types/DrinkRecipe"
+
+const fakeImageUrl =
+  "https://www.liquor.com/thmb/AiiScF_3Lk7T9UhtN1nJzKoFU20=/720x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/pumpkin-buck-720x720-primary-bf4b55c8a12b4039be883ec68b9108f5.jpg"
+
+const pumpkinBuck: DrinkRecipe = {
+  name: "Pumpkin Buck",
+  imageUrl: fakeImageUrl,
+  measuredIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Applejack",
+      unit: "oz",
+      unitAmount: 1.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Pumpkin Purée",
+      unit: "oz",
+      unitAmount: 15.0,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Water",
+      unit: "Quart",
+      unitAmount: 3.0,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Maple Syrup Grade A",
+      unit: "oz",
+      unitAmount: 3.0,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Ginger",
+      unit: "Cup",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+  ],
+  numberedIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Numbered,
+      ingredient: "Lemon",
+      amount: 6,
+      alternates: [],
+    },
+
+    {
+      type: DrinkRecipeIngredientType.Numbered,
+      ingredient: "Whole Cloves",
+      amount: 4,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Numbered,
+      ingredient: "Star Anise Pod",
+      amount: 3,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Numbered,
+      ingredient: "Cardamom Pod",
+      amount: 4,
+      alternates: [],
+    },
+  ],
+  portionlessIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Cinnamon",
+      suffix: "Stick",
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      prefix: "Split",
+      ingredient: "Vanilla Bean",
+      alternates: [],
+    },
+  ],
+  garnishIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      prefix: "Candied",
+      ingredient: "Ginger",
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      prefix: "Grated",
+      ingredient: "Nutmeg",
+      alternates: [],
+    },
+  ],
+  stepsToMake: [
+    "To make Pumpkin Soda:",
```

Comment:
I don't think this one should be in here. If we decide to add a label overhead the steps to make, this bit will look redundant. Also, if we number the steps to make a drink, this item will get the `1.`